src/workers/extractEmissions.ts

```diff
@@ -93,6 +99,13 @@ const extractEmissions = new DiscordWorker<JobData>(
             type: JobType.Initiatives,
           },
         },
+        {
+          name: 'askDiff ' + companyName,
+          queueName: 'askDiff',
+          data: {
+            ...base.data,
+          },
+        },
```

Comment:
Do we really want to run `askDiff()` in parallel to the main data extraction? This should most likely be removed.